### PR TITLE
relayer.md added

### PR DESCRIPTION
**Pull request by maksym-nyrka**

### CODE REVIEW

File: src/developer-documentation/relayer/relayer.md [hunk 1]
```diff
@@ -0,0 +1,156 @@
+## ETH Light Client & Relayer setup & usage example
+
+For now ETH Light Client is integrated only with Brooklyn.
```
Comment (uchu) [inline src/developer-documentation/relayer/relayer.md:3]: @maksym-nyrka Let's make it more clear that these instructions are for Brooklyn (and to mention it's a dev chain), but the process will be similar to Sydney and mainnet.

As it's easy to get lost between chains. Maybe just to make it a bold header or kinda disclaimer.

**Review by uchu — CHANGES_REQUESTED** [review]
LGTM, except that we need better emphasis that it currrently works on Brroklyn.

File: src/developer-documentation/relayer/relayer.md (hunk 1 above)
Comment (uchu) [reply src/developer-documentation/relayer/relayer.md:3]: We just don't mention Brooklyn anywhere, so maybe a good place to make some explanation here.

**Review by uchu — APPROVED** [review]
LGTM

**Review by akorchyn — APPROVED** [review]
Great job! Thanks

**Pull request merged**